data/na/swagger/1-0-5.yaml

```diff
@@ -2370,6 +2555,18 @@ definitions:
           $ref: '#/definitions/Detail'
       validation:
         $ref: '#/definitions/CardValidation'
+  RecurringPaymentErrorResponse:
+    properties:
+      code:
+        type: integer
+        format: int32
```

Comment:
why do we say int32 here but integer elsewhere? should be consistent

Reply:
not sure, copied it from elsewhere.